apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -1134,11 +1141,13 @@ handle_info({force_update_presence, LUser}, StateName,
     NewStateData =
     case StateData#state.pres_last of
         #xmlel{name = <<"presence">>} ->
-            PresenceEl = ejabberd_hooks:run_fold(
+            Stanza = mongoose_stanza:from_kv(packet, StateData#state.pres_last),
```

Comment:
Why `packet` field here? I saw that for the same hook in the same module you used `element` field.